applications/nrf5340_audio/src/utils/board_init.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _BOARD_H_
```

Comment:
"Board" is quite generic.

Reply:
Agreed will rename as nrf5340_dk_init.[c,h]